### DISCUSSION TITLE
capture_array() for monochrome image sensor

### DISCUSSION THREAD

**Discussion by waikeat2397** [raspberrypi/picamera2#796] [Q&A]
I read in the picamera2 documentation that capture_array function will  return a three-dimensional numpy array. How about if I am using a monochrome image sensor? I tried and I still got a three-dimensional numpy array. Is there any alternative way that I can retrieve a one-dimenstional numpy array using capture_array function? Or i need to convert the image into grayscale manually using opencv? Convert it manually using opencv will cause the value of the pixel change?

**Comment by davidplowman** [comment]
You should find that all three channels are the same. An alternative might be to ask for a YUV420 image and then discard the final 1/3 of the rows, leaving you with just the Y component (effectively, greyscale).
